.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+sudo: false
+node_js:
+  - 6
+os:
+  - linux
+  - osx
+before_install:
+  - pip install --user cpplint
```

Comment:
Sorry for the delay. Sadly we need `--user` for GNU/Linux, otherwise it fails there. I'll add a conditional for it.